### PR TITLE
Update changelog for #10350

### PR DESCRIPTION
**Pull request by korycins**
I want to merge this change because it adds missing change log for already merged feature parts

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

**Review by fowczarek — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,16 @@
...
+- Improve support for handling transactions - #10350 by @korycins
+  - Add new type `GrantedRefund`. It defines the amount of the refund, granted for the order.
+  - Add new mutation `orderGrantRefundCreate` and `orderGrantRefundUpdate` responsible for managing the granted refunds attached to the order.
+  - Add new fields to Order type:
+    - `grantedRefunds` list of `GrantedRefund` assigned to the order.
+    - `totalGrantedRefund` total amount of all granted refunds
+		- `totalRefunded` total amount of refund
+    - `totalPendingRefund` total amount of pending refund
+    - `totalRemainingGrant` the difference amount between granted refund and the amounts that are pending and refunded.
 ### Other changes
```
Comment (IKarbowiak) [inline CHANGELOG.md:16]: Lack of space before `### Other changes`

File: CHANGELOG.md [hunk 2]
```diff
@@ -3,7 +3,16 @@
 All notable, unreleased changes to this project will be documented in this file. For the released changes, please visit the [Releases](https://github.com/mirumee/saleor/releases) page.
 
 # 3.8.0 [Unreleased]
-
+### Highlights
+- Improve support for handling transactions - #10350 by @korycins
```
Comment (IKarbowiak) [inline CHANGELOG.md:7]: Here also should be an empty line after `Highlights`

File: CHANGELOG.md (hunk 1 above)
Comment (korycins) [reply CHANGELOG.md:16]: Solved in: https://github.com/saleor/saleor/pull/10687

File: CHANGELOG.md (hunk 2 above)
Comment (korycins) [reply CHANGELOG.md:7]: Solved in: https://github.com/saleor/saleor/pull/10687

**Pull request merged**